client/styles/index.scss

```diff
@@ -3,6 +3,7 @@
 @import './events-list.scss';
 @import './add-event-container.scss';
 @import './add-event-form.scss';
+@import './react-geosuggest.scss';
```

Comment:
you should import this where you are importing the geosuggest module, in `AddGeoSuggestInput.jsx` (as I did with the timepicker)